src/libtriton/arch/x86/x8664Cpu.cpp

```diff
@@ -15,6 +15,7 @@
 #include <triton/immediate.hpp>
 #include <triton/x8664Cpu.hpp>
 
+#include <iterator>
```

Comment:
Why this include?

Reply:
I just forgot it, part of some tests. It should be removed in the last commit?